darkwolf/theme.json

```diff
@@ -0,0 +1,8 @@
+{
+    "name": "darwolf",
+    "version": "5.0",
+    "description": "Default phpMyAdmin theme",
+    "author": "phpMyAdmin developers",
```

Comment:
I've asked the author in https://github.com/phpmyadmin/themes/pull/51#issuecomment-727464939 if there's a better name or alias to use here. I don't like to claim authorship of something we didn't make :)